scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -1005,6 +1018,16 @@ def _create_build_dir_link(self, links_dir_path, instance):
 
         self.link_dir_counter += 1
 
+    def create_path_with_script(self, script_name,platform):
+
+        if platform.arch == "riscv32":
+            script_path = os.path.join(ZEPHYR_BASE, "boards", "riscv", platform.name, "support", f"{script_name}.sh")
+        else:
+            script_path = os.path.join(ZEPHYR_BASE, "boards", platform.arch, platform.name, "support", f"{script_name}.sh")
```

Comment:
only for linux like system?

Reply:
i made fix for selecting os, what concerns do you have?

Reply:
the scripts itself is hard to control and prove, it depends on many other things. would this be better that we use board overly for tests.yaml? and enable scripts only for given board, instad of adding more and more in test.yaml file